composer.json

```diff
@@ -2,15 +2,16 @@
 	"name": "automattic/sensei-lms",
 	"description": "A learning management plugin for WordPress, which provides the smoothest platform for helping you teach anything.",
 	"require-dev": {
-		"php": "^7.2 || ^8",
-		"bamarni/composer-bin-plugin": "^1.4",
-		"dealerdirect/phpcodesniffer-composer-installer": "0.7.1",
-		"phpcompatibility/phpcompatibility-wp": "2.1.2",
+		"php": "^7.2.5 || ^8",
+		"bamarni/composer-bin-plugin": "1.8.2",
+		"dealerdirect/phpcodesniffer-composer-installer": "0.7.2",
+		"dms/phpunit-arraysubset-asserts": "^0.4.0",
```

Comment:
This is a polyfill for `assertArraySubset`. The assert was deprecated in PHPUnit 8 with no replacement.